tool/generate-specs.sh

```diff
@@ -50,6 +50,14 @@ done
   generate_spec "." "notifications"
 )
 
+for spec in packages/nextcloud/lib/src/api/*.openapi.json; do
+  name="$(basename "$spec" | cut -d "." -f 1)"
+  if [ -f "tool/patches/$name.json" ]; then
```

Comment:
I'd change the format to `tool/patches/$appName/$patchName.json` this way we can support multiple patches per spec.

This way we can keep the patches themselves smaller and easier to understand.

Reply:
I wanted to keep it simple for the beginning, but it makes sense to already have this.